hooks/post_gen_project.py

```diff
@@ -0,0 +1,15 @@
+import os
+import sys
+
+REMOVE_PATHS = [
+    '{% if cookiecutter.use_hpack == "y" %} {{ cookiecutter.project_name }}.cabal {% endif %}',
+    '{% if cookiecutter.use_hpack != "y" %} package.yaml {% endif %}',
```

Comment:
Unsure if I should do case insensitive match here

Reply:
I agree that would be better. I think, more importantly, it isn't ideal in the sense that it's not obvious which strings are allowed, and one can enter nonsense which will still pass. Maybe something like [Choice Variables](https://cookiecutter.readthedocs.io/en/latest/advanced/choice_variables.html) functionality might work better there, eg. it could be something like:

```
Select package description format:
1 - Cabal
2 - HPack
Choose from [1], 2:
```

What do you think? It's a bit more wordy, but at least it is obvious that it's a choice between two.

Reply:
I agree because the default is more obvious. I am just hoping I can make `{{project_name}}.cabal` or `package.yaml (hpack)` or something

Reply:
Handled in e125757cd58bc0117c0b9dd1fa8ee31f70713f38